### PR TITLE
Make smoke tests run on prod or staging

### PR DESCRIPTION
**Pull request by Ian-Nicholls**
**WHAT**

Add an end to the ifs and make STAGING need to equal 1 to do staging stuff

**WHY** 

Else it doesnt run it

### CODE REVIEW

File: spec/system/admin/shared_context.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 RSpec.shared_context "admin", shared_context: :metadata do
   before(:all) do
-    if ENV["STAGING"] 
+    if (ENV["STAGING"] = 1)
```
Comment (colinbm) [inline spec/system/admin/shared_context.rb:3]: Comparison would need to be `==` - you're assigning inline here (surprised rubocop doesn't yell?)

However I'd leave this as it was, and remove `STAGING: 0` above... - `0` is still _truthy_ in ruby, but also I think these values come in as strings regardless, so it's `"0"` (truthy in most languages I think [except PHP])

File: spec/system/signup/signup_spec.rb [hunk 2]
```diff
@@ -9,10 +9,11 @@
 
     test_email = gmail.account_email.gsub(/@/, "+#{Time.now.to_i}@")
 
-    if ENV["STAGING"] 
+    if (ENV["STAGING"] = 1)
```
Comment (colinbm) [inline spec/system/signup/signup_spec.rb:12]: Same here - better without `= 1`

File: smoke-tests-concourse.yml [hunk 3]
```diff
@@ -124,6 +124,7 @@ jobs:
         GOOGLE_API_TOKEN_DATA: "((GOOGLE_API_TOKEN_DATA))"
         RADIUS_KEY: "((RADIUS_KEY))"
         RADIUS_IPS: "((RADIUS_IPS))"
+        STAGING: 0
```
Comment (colinbm) [inline smoke-tests-concourse.yml:127]: As mentioned, remove this

File: spec/system/signup/signup_spec.rb [hunk 4]
```diff
@@ -9,10 +9,11 @@
...
       emailaddress="signup@staging.wifi.service.gov.uk"
     else
       emailaddress="signup@wifi.service.gov.uk"
```
Comment (colinbm) [inline spec/system/signup/signup_spec.rb:15]: Suspect rubocop fixes didn't make the commit? It won't like the lack of spaces around `=`

File: spec/system/signup/signup_spec.rb (hunk 2 above)
Comment (colinbm) [inline spec/system/signup/signup_spec.rb:12]: Also, no need for `()` around condition - `if ENV["STAGING"]` is sufficient - another one rubocop should yell about.

**Review by colinbm — CHANGES_REQUESTED** [review]
Hot tip - `rubocop -a` should auto-fix most if not all of these.

File: spec/system/signup/signup_spec.rb (hunk 2 above)
Comment (Ian-Nicholls) [reply spec/system/signup/signup_spec.rb]: done

**Review by colinbm — APPROVED** [review]

**Pull request merged**